src/main/java/com/laioffer/trademarket/entity/User.java

```diff
@@ -12,11 +12,14 @@ public class User implements Serializable {
 
     @Id
     private String username;
+    private String password;
+
+    private String firstname;
+    private String lastname;
+
     @JoinColumn(unique = true)
```

Comment:
Do we want Foreign Key here or unique constraint? I searched to find out @Column(unique=true) for single column constraint and JoinColumn for Foreign Key.